imsv-docs-docusaurus/openapi/endpoints/kyc/models/retrieve-kyc-statement-response.yaml

```diff
@@ -30,3 +30,7 @@ properties:
     description: A list of evidence supporting the claimed identity. At least one piece of evidence is required.
     items:
       $ref: './evidence.yaml'
+  region:
+    type: string
+    description: An Alpha3 (ISO-3166-1) country code representing the country in which the user is being verified.
```

Comment:
Just out of curiosity, did we decide to use `country` as the final wording? I remember we were talking about places like EU which is not a country so I was wondering if the language here needs to reflect that

Reply:
that's a great question, I am not too sure about it. @rb455 any thoughts?

Reply:
I also think it should ideally be "country". The only problem is that using "region" where we actually mean "country" is entrenched a bit:
- kycRegion in spending prerequisites  - this will need to be renamed
- region field on `KycProfile`, `KycSession`, `KycStatement` models - the field can stay in those, but we just need to add another field - 'country', which will be populated with the same value as region.

All that is doable just need to make a judgement if its worth it at this stage

Reply:
@catluin @ncjones a discussion was held and the decision was made to stick to 'region' as something more flexible. In case we want to distinguish users by, say, US states in the future, that will leave us this flexiibility. https://www.notion.so/immersve/KYC-regions-discussion-2ca91fe2cc124d32b78bca1c827ad1c2?pvs=4#9d88627721f841f2a8e9930294b09d53